Cubical/Foundations/HLevels.agda

```diff
@@ -38,6 +39,11 @@ isOfHLevel 0 A = isContr A
 isOfHLevel 1 A = isProp A
 isOfHLevel (suc (suc n)) A = (x y : A) → isOfHLevel (suc n) (x ≡ y)
 
+-- a re-indexing of `isOfHLevel` which is consistent with standard notation,
+--  in the HoTT book this is `is-n-type(X)`
+isTrunc : ℕ₋₂ → Type ℓ → Type ℓ
+isTrunc n = isOfHLevel (2+ n)
```

Comment:
I'm also unsure about the inclusion of this definition. It is nice to have in conjunction with `∥_∥_` (and in the future, `isConn`, `isConnMap`, `isTruncMap`), but every fact about it is currently stated in terms of h-levels. Should we therefore have a file that consists solely of re-indexed versions of theorems in `HLevels.agda`? (e.g. `isTruncSuc`, `isTruncPath`, `isTruncPi`, etc.) This seems excessive, but it is in the spirit of @mortberg's suggestion in #238 to "work internally with ℕ but also expose standard interfaces with `ℕ₋₂`".

As-is, using it is clunky (e.g. in the definition of `decode-fun` in `Truncation.Properties` there are terms like: `isOfHLevelSuc (2+ n) (isTrunc∥∥ n A)`).

Reply:
I don't know... I'm personally fine with working with h-levels, but it's nice to have this definition as documentation to reduce confusion for people who read the HoTT book.

Reply:
And yeah, I don't think we should redo the HLevels file shifted. I was mainly thinking that we could provide the ℕ₋₂ interface for n-truncation. But maybe it will be hard to use because all theory is stated for h-levels?

Reply:
For now I think I should remove `isTrunc` from this PR and make an issue asking the community about this. `∥_∥_` is nice to have in theory, but mostly unusable without similarly indexed lemmas about hlevels / truncation levels.

Reply:
Sure, makes sense to me